### ISSUE TITLE
IOS gets wrong token

### ISSUE THREAD

**Issue by ahar** [phonegap/phonegap-plugin-push#2604]
### Expected Behaviour
To get token with length 152 characters. 

### Actual Behaviour
Gets token with length 64 characters, not formatted correctly( 0d87503e0e0171501a5d10fa4e1405077ac08eb829631ad358a7f6f49c0c6cb9 )
### Reproduce Scenario (including but not limited to)
Create new project, and try to get token.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
ios 12.0.1

#### Cordova CLI version and cordova platform version

    cordova --version 8.0.0
    cordova platform version ios 4.5.5                     

#### Plugin version
2.1.2

**Comment by macdonst** [comment]
@ahar it looks like your GoogleServices pList file is not being delivered properly during the building of your app.

**Comment by macdonst** [comment]
@ahar it looks like your google services file is not being picked up at build time as that is a APNS style registration ID.

**Comment by ygyg70** [comment]
My GoogleServices pList is in the resources folder.
For the same app store build, some users get a long token and some get a short token
I a now trying to find a phone that has a bad token, as all the ones I have access to have a good token therefore I cannot troubleshoot.

**Comment by ygyg70** [comment]
IS_GCM_ENABLED is set to YES in GoogleService-info.plist
Is there any other place I should look into?
What would you look for in config.xml? As far as I understand with version 2.2.3 there isn't any configuration for this plugin that goes there

**Comment by macdonst** [comment]
@ygyg70 yeah, it does. Read: https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/INSTALLATION.md#installation-requirements fully.

**Comment by ygyg70** [comment]
Just checked my config.xml, and I did ave the resource lines in the right place.
To recap, things have been working fine, both iOS and Android generating the FCM tokens, then at some point (few months ago) some iOS devices started to generate the 64 character tokens. All using the same app build.
I am trying to find a device that has the short token for further investigation, as all the devices I have work correctly.

**Comment by ygyg70** [comment]
Config.xml attached, with app name masked
[config.xml.txt](https://github.com/phonegap/phonegap-plugin-push/files/2559483/config.xml.txt)

**Comment by skimi** [comment]
64 characters, could it be an APN token? The opposite as the issue #2542 were FCM token get generated instead of APN.
